### ISSUE TITLE
queries with aliases containing at least one uppercase letter do not work properly with filters

### ISSUE THREAD

**Issue by AIEAJN** [MrThearMan/graphene-django-query-optimizer#121]
### I have read the docs thoroughly before making this bug report.

- [X] Yes

### I have read through other open issues, and my issue is not a duplicate.

- [X] Yes

### What version of the library you are using?

0.8.4

### Which python version are you using?

3.12

### What operating system are you on?

Windows

### Description

When you use aliases containing uppercase letters, the filters don't work, returning an incorrect answer.
```
query {
  enven: pagedBuildings(name: "environmentphonesay"){
    edges {
      node {
        id
        name
        streetAddress
        
      }
    }
  }  
  
  ofTen:pagedBuildings(name: "oftenIcell"){
    edges {
      node {
        id
        name
        streetAddress
      }
    }
  }
}
```

**Comment by AIEAJN** [comment]
![Screenshot 2024-07-03 132308](https://github.com/MrThearMan/graphene-django-query-optimizer/assets/47372918/8648cd33-c26e-44e3-af0c-cc77e3a883e2)

**Comment by MrThearMan** [comment]
Ok, this was a small error with converting the top-level alias to snake case when retrieving filter information. Should be fixed in [0.8.5](https://github.com/MrThearMan/graphene-django-query-optimizer/commits/main/#diff-a871d11b7bfb07650183b83348bbc6b8373944be2167e31cf3c591f3c29cb9c8).
